src/cbor.h

```diff
@@ -147,6 +147,7 @@ typedef enum CborError {
     CborErrorUnsupportedType,
 
     /* errors in converting to JSON */
+    CborErrorJsonNotImplemented,
```

Comment:
Please add it after the other Json entries, so as to not change the values of the existing enums.